src/node_perf.cc

```diff
@@ -385,6 +391,168 @@ void Timerify(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(wrap);
 }
 
+// Event Loop Timing Histogram
+namespace {
+static void ELDHistogramMin(const FunctionCallbackInfo<Value>& args) {
```

Comment:
Just a note, `namespace {` and `static` do pretty much the same thing, so you can drop `static` here and below, if you like.